coding-the-legislation/35_periods.md

```diff
@@ -1,25 +1,103 @@
 # Periods
 
+A period can be a month, a year, `n` successive months or `n` successive years.
```

Comment:
Right now indeed, a period can not be `ETERNITY`. Only a variable can have `ETERNITY` as its definition period.